pom.xml

```diff
@@ -61,10 +61,12 @@
     <properties>
         <!-- Dependency Versions -->
         <bouncycastle.version>1.46</bouncycastle.version>
+        <cobertura.version>2.1.1</cobertura.version>
         <java-mail.version>1.6.1</java-mail.version>
-        <junit.version>5.1.0</junit.version>
+        <junit.version>1.2.0</junit.version>
```

Comment:
This should be switched back to `5.2.0` since that's the version of junit that we are using.  I would suggest adding a second `junit-platform.version` property like they do in this example: https://github.com/junit-team/junit5-samples/blob/r5.2.0/junit5-jupiter-starter-maven/pom.xml